portality/templates/doaj/toc.html

```diff
@@ -36,380 +42,356 @@
-                #}
-                {#
-                {% if bibjson.archiving_policy %}
-                    {% if bibjson.archiving_policy.get("policy") or bibjson.archiving_policy.get("url") %}
-                    <p>Content digitally archived in:
-                        {% if bibjson.archiving_policy.get("url") %}
-                            <a href="{{bibjson.archiving_policy.get('url')}}">{{bibjson.flattened_archiving_policies|join(", ")}}</a>
-                        {% else %}
-                            {{bibjson.flattened_archiving_policies|join(", ")}}
-                        {% endif %}
-                    </p>
-                    {% endif %}
-                {% endif %}
-                #}
-                {% if bibjson.article_statistics %}
-                    {% if bibjson.article_statistics.get("statistics") or bibjson.article_statistics.get("url") %}
-                    <p>Article-level metrics:
-                        {% set c = bibjson.article_statistics.get("statistics", "uncertain") %}
-                        {% if bibjson.article_statistics.get("url") %}
-                            <a href="{{bibjson.article_statistics.get('url')}}">{{TYN.get(c, c)}}</a>
-                        {% else %}
-                            {{TYN.get(c, c)}}
-                        {% endif %}
-                    </p>
-                    {% endif %}
+                {% if bibjson.institution %}
+                <dt>Society or institution</dt>
+                <dd>{{bibjson.institution}}{% if bibjson.institution_country %}, {{ bibjson.institution_country }}{% endif %}</dd>
                 {% endif %}
 
-                {% if bibjson.deposit_policy %}
-                    <p>Deposit policy registered in: {{bibjson.deposit_policy|join(", ")}}</p>
+                {% if bibjson.language %}
+                <dt>Manuscripts accepted in</dt>
+                <dd>{{bibjson.language_name()|join(", ")}}</dd>
                 {% endif %}
-
-                {% if lic %}
-                    {% if lic.get("embedded") %}
-                        <p>Electronically embedded licenses? {{TYN.get(lic.get("embedded"), lic.get("embedded"))}}</p>
-                    {% endif %}
+              </dl>
+            </article>
+
+            <article class="card card--with-icon">
+              <span data-feather="tag" aria-hidden="true"></span>
+              <dl>
+                {% for path in bibjson.lcc_paths() %}
+                {% if loop.index0 == 0 %}<dt>LCC subjects <a href="https://www.loc.gov/catdir/cpso/lcco/" target="_blank" rel="noopener"><span data-feather="help-circle" aria-hidden="true"></span></a></dt>{% endif %}
+                <dd>{{ path }}</dd>
+                {% endfor %}
+
+                {% if bibjson.keywords %}
+                <dt>Keywords</dt>
+                <dd>
+                  {% for keyword in bibjson.keywords %}
+                  <a href="">{{ keyword }}</a>{% if forloop.last %}{% else %}, {% endif %}
```

Comment:
it might be `loop.last` in jinja2 (2.10.3) . Also, this is better represented as `{{ ", " if not loop.last }}`

Reply:
My bad -- this is taken directly from my Jekyll templates (uses Liquid for templating). Since I couldn’t visually see how it renders, I didn’t realise it didn’t work!